### PR TITLE
Modifies spectral instruments to make retaliation more viable. 

### PR DESCRIPTION
**Pull request by Pinkufeck**
## About The Pull Request

Reduces the damage dealt by the spectral trombone, saxophone and trumpet and adds a message warning of the change they cause

## Why It's Good For The Game

In their current state the spectral instruments transform people into 4 hits, the first two hits dealing enough stam damage that it slows to point where escape isn't possible. 
Since it's also rare most people don't know of it'll transform you into a skeleton
In order to allow for more viable retaliation against the transformation into a skeleton this PR reduces the stamina damage dealt and prints a message for the target that hints them towards the transformation. 

## Changelog

:cl:Somepan
add: Added a message upon being hit by any spectral instruments
balance: Reduced the stamina damage dealt by spectral instruments from 25 to 18 making it transform after 6 hits instead of 4
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
I'd personally go for 20 as a middle ground?

Remember that this is reduced by armor.

**Comment by Pinkufeck** [comment]
> This is... incorrect? It currently takes 4 hits to spookify someone, and your PR makes the damage 16, not 15.

My bad on that, in my head i somehow thought 25*3>95
As for the damage, changed it and forgot to change, oops.

**Comment by Pinkufeck** [comment]
> I'd personally go for 20 as a middle ground?
> 
> Remember that this is reduced by armor.

20 would make it a 5 hit on unarmored people, i don't think the change would be significant enough especially since skeletons tend to gang up on people to transform them. 
And most people don't have armor, i aim to allow retiliation, or flight for ANYONE, not just people who have armor.

**Comment by ATH1909** [comment]
> > I'd personally go for 20 as a middle ground?
> > Remember that this is reduced by armor.
> 
> 20 would make it a 5 hit on unarmored people, i don't think the change would be significant enough especially since skeletons tend to gang up on people to transform them. And most people don't have armor, i aim to allow retiliation, or flight for ANYONE, not just people who have armor.

do you knock someone into damage slowdown with 20 stamina hit? if not, they can just yakety sax away after your first hit, no?

**Comment by Helg2** [comment]
> 20 would make it a 5 hit on unarmored people, i don't think the change would be significant enough especially since skeletons tend to gang up on people to transform them. And most people don't have armor, i aim to allow retiliation, or flight for ANYONE, not just people who have armor.

You just sound salty now. Skill issue.

File: code/datums/elements/spooky.dm [hunk 1]
```diff
@@ -38,8 +38,9 @@
 		C.set_jitter_if_lower(70 SECONDS)
 		C.set_stutter(40 SECONDS)
 		if((!istype(H.dna.species, /datum/species/skeleton)) && (!istype(H.dna.species, /datum/species/golem)) && (!istype(H.dna.species, /datum/species/android)) && (!istype(H.dna.species, /datum/species/jelly)))
-			C.adjustStaminaLoss(25) //boneless humanoids don't lose the will to live
+			C.adjustStaminaLoss(16) //boneless humanoids don't lose the will to live
 		to_chat(C, "<font color='red' size='4'><B>DOOT</B></font>")
+		to_chat(C, "<span class='robot'><font size=4>You're feeling more bony.</font></span>")
```
Comment (Fikou) [inline code/datums/elements/spooky.dm:43]:
```suggestion
		to_chat(C, "<span class='robot'><font size='4'>You're feeling more bony.</font></span>")
```

File: code/datums/elements/spooky.dm [hunk 2]
```diff
@@ -38,8 +38,9 @@
 		C.set_jitter_if_lower(70 SECONDS)
 		C.set_stutter(40 SECONDS)
 		if((!istype(H.dna.species, /datum/species/skeleton)) && (!istype(H.dna.species, /datum/species/golem)) && (!istype(H.dna.species, /datum/species/android)) && (!istype(H.dna.species, /datum/species/jelly)))
-			C.adjustStaminaLoss(25) //boneless humanoids don't lose the will to live
+			C.adjustStaminaLoss(16) //boneless humanoids don't lose the will to live
```
Comment (Fikou) [inline code/datums/elements/spooky.dm:41]: 18 should be fine i think, you have enough time to retaliate

**Comment by Mechar418** [comment]
Becoming a skeleton brings only the advantages of the race + sounds like a major skill issue

**Comment by Pinkufeck** [comment]
> You just sound salty now. Skill issue.
I'm the first one to use it on others, couldn't give less of a fuck.

**Pull request merged**